props/saber_fett263_buttons.h

```diff
@@ -4533,7 +4542,11 @@ SaberFett263Buttons() : PropBase() {}
             return true;
 #endif
           } else {
-            DoIgnition();
+            if (!CheckInteractivePreon()) {
```

Comment:
switch if and else branches around and remove the "!"

Reply:
moved to helper function and reversed